docker/mostro/entrypoint.sh

```
@@ -0,0 +1,29 @@
+  useradd -M -u $USERID -g $GROUPID mostro
+  chown -R $USERID:$GROUPID /home/mostro
+fi
+
+if [ $(echo "$1" | cut -c1) = "-" ]; then
+  echo "$0: assuming arguments for mostrod"
+
+  set -- mostrod "$@"
+fi
+
+if [ "$1" = "mostrod" ] || [ "$1" = "mostro-cli" ]; then
+  echo "Running as mostro user: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)